cEP-0005.md

```diff
@@ -104,7 +104,7 @@ class RedundancyBear(LocalBear):
     # ... and detect code clones.
     DETECT_ASPECTS = {Root.Redundancy.Clone}
```

Comment:
@Techievena You should really change this to our new meta `bearclass` style

And you should also change it to use new `languages=` keyword in `class` definition args according to https://github.com/coala/coala/issues/4111